src/sec_certs/sample/document_state.py

```diff
@@ -0,0 +1,61 @@
+from dataclasses import dataclass
+from pathlib import Path
+
+from sec_certs.serialization.json import ComplexSerializableType
+
+
+@dataclass
+class DocumentState(ComplexSerializableType):
```

Comment:
DocumentState should likely be also used in the FIPS certificate for the security policy, right? Is now a good time to add it or no?

Reply:
See https://github.com/crocs-muni/sec-certs/issues/464.

Reply:
Oh, I even saw that but forgot.